dist/components/address/address.js

```diff
@@ -15,6 +15,10 @@ var AddressComponent = (function (_super) {
     AddressComponent.prototype.allowMultiple = function () {
         return false;
     };
+    //return true if component sets an array in setValue...
+    AddressComponent.prototype.isMultipleValueComponent = function () {
+        return true;
```

Comment:
I also don't like that ```isMultipleValueComponent``` means "This component stores its values as an array", whereas ```allowMultiple``` means "This component should allow the multiple option to be checked. I am pretty certain we can combine these.